Compilation warning: C++ comments are not allowed in ISO C90

* **Version**: 1.47.0
* **Platform**: Windows 10 using MinGW

Hello, I get the following warning in Windows:
```
include/uv/win.h:48:1: warning: C++ style comments are not allowed in ISO C90
   48 | // Disable the typedef in mstcpip.h of MinGW.
      | ^
```
I noticed that there are 2 instances in win.h with double slash comments:
https://github.com/libuv/libuv/blob/1dd0ab13154ccad38839807fe4909469f7164e25/include/uv/win.h#L48
https://github.com/libuv/libuv/blob/1dd0ab13154ccad38839807fe4909469f7164e25/include/uv/win.h#L73

This is not a critical issue, but I image that it will break some builds when using older compilers.

Thanks,
Trevor

--- a/include/uv/win.h
+++ b/include/uv/win.h
@@ -45,7 +45,7 @@ typedef struct pollfd {
 #endif
 
 #include <mswsock.h>
-// Disable the typedef in mstcpip.h of MinGW.
+/* Disable the typedef in mstcpip.h of MinGW. */
 #define _TCP_INITIAL_RTO_PARAMETERS _TCP_INITIAL_RTO_PARAMETERS__AVOID
 #define TCP_INITIAL_RTO_PARAMETERS TCP_INITIAL_RTO_PARAMETERS__AVOID
 #define PTCP_INITIAL_RTO_PARAMETERS PTCP_INITIAL_RTO_PARAMETERS__AVOID
@@ -70,7 +70,7 @@ typedef struct pollfd {
 # define S_IFLNK 0xA000
 #endif
 
-// Define missing in Windows Kit Include\{VERSION}\ucrt\sys\stat.h
+/* Define missing in Windows Kit Include\{VERSION}\ucrt\sys\stat.h */
 #if defined(_CRT_INTERNAL_NONSTDC_NAMES) && _CRT_INTERNAL_NONSTDC_NAMES && !defined(S_IFIFO)
 # define S_IFIFO _S_IFIFO
 #endif

